### ISSUE TITLE
Use value of alpha from Von Caemmerer (2009) in FcVB (calculation of J)

### ISSUE THREAD

**Issue by rpaperez** [VEZY/PlantBiophysics.jl#51]
alpha is taken from Duursma (PlantEcophys), which is abs.(1-f)/2 (cf Van Caemmerer 2009). But in PlantBiophysics, the absorbed PPFD is used and not the incident PPFD. So alpha should be (1-f)/2, value of 0.42 instead of 0.24
Planbiophysis used theta=0.7 from Caemmerer whereas used theta=0.85. Should use both value on Von Caemmerer for alpha and theta. 
Also should detail in PlantBiophysics the use of aPPFD and not incident PPFD. Calibration of photosynthesis parameters should use aPPFD and not incident PPFD (aPPFD=0.85*iPPFD)

https://github.com/VEZY/PlantBiophysics.jl/blob/26774896863e8f253e69ce4b560b06d6570cba36/src/processes/photosynthesis/FvCB.jl#L32

Von Caemmerer 2009:
DOI: [10.1007/978-1-4020-9237-4_9](http://dx.doi.org/10.1007/978-1-4020-9237-4_9)

**Comment by VEZY** [comment]
α and θ are also taken from Medlyn et al. (2007) I think.
I don't remember our discussions with Julien very well, but in Von Caemmerer (2009) the absorbed light I2 is computed as follows in equation 9.16:

$I2 = I · abs · (1 − f ) / 2$

with $I$ the incident light to the leaf, $abs$ is the absorptance of the leaf (commonly 0.85), f is a correction factor for light quality (~0.15) and the division by 2 is because they assume half the light absorbed needs to reach each photosystem.

So I think our code is right, because in our function α is the quantum yield of electron transport, not the absorptance. The absorptance should be used earlier by the light interception model to compute the aPPFD from the incident light.

@JulienLamour do you confirm?

**Comment by JulienLamour** [comment]
(1-f)/2 is the quantum yield of absorbed light and is 0.425 in Von Caemmerer (2009). In Caemmerer et al. (2009), the electron transport curvature factor, theta, is 0.7 and the leaf absorptance is 0.85. Some people refer to the quantum yield of incident light, i.e 0.425 * 0.85 = 0.36. Medlyn et al. (2002) use a different parametrization, i.e theta = 0.9 and a quantum yield of absorbed light of 0.375. I.e a quantum yield of incident light of 0.30 with a leaf absorptance that they set at 0.8. I don't know where Duursma et al. found the 0.24 values but it looks very low.  In all cases, Theta, quantum yield and leaf absorptance should be taken from the same study (do not mix and match). This is because the estimation of these parameters from light curves is not independent. Therefore, I do not recommend to set alpha = 0.24 together with theta = 0.7.

@VEZY I2 is not the absorbed light but the light that reaches photosystem 2 and is used for photochemistry. The absorbed light is I*abs. I see that you use the absorbed PPFD as an input variable to the FvCB model, so in my opinion you should use theta = 0.7 and alpha = 0.425 (Caemmerer 2009) or theta = 0.9 and alpha = 0.375.

**Comment by VEZY** [comment]
@JulienLamour thanks for your answer! You're right I mixed different sources for the values of α and θ. I took the value of α from `plantecophys` (0.24, but there `θ=0.85`), and the value of θ from Caemmerer (2000).

Yes you're right, absorbed light is $I \cdot abs$, not $I2$.

I opened a PR here: #60 and am about to merge it. I checked the outputs on the paper evaluation, and it didn't change anything. It's probably because the error was compensated in the fitting step, but at least now it's clearer, and it helped me find an issue in the way we read the files, and fix the way we simulated with LeafGasExchange.jl.
